api-server/src/main/java/io/enmasse/api/v1/http/HttpHealthService.java

```diff
@@ -28,4 +30,21 @@ public HealthResponse(int status, String message) {
             this.message = message;
         }
     }
+
+    @GET
+    @Produces({MediaType.TEXT_HTML})
+    @Path("metrics")
+    public Response getMetrics() {
+        String name = this.getClass().getPackage().getImplementationTitle();
+        String version = this.getClass().getPackage().getImplementationVersion();
```

Comment:
Is this the name and version of the component? If so, I would prefer to hardcode the name here, and get version from the image environment. Then we don’t need the extra maven stuff either right?